### PR TITLE
Add django env aa1 to implement aa1

### LINKED ISSUE #3: Django Template for TASTIC engine
**Describe the feature or change you'd like**
A basic Django project environment should be set up.

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no environment for working with Django. #3 

**What is the new behavior (if this is a feature change)?**
- Now there is an environment to work with Django. Closes #3 

**Other information**:
- Ref: 🐍

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM 😘

**Comment by schettn** [comment]
What you up to? Should this supposed to be a wagtail instance or just pure django?
@pinterid

**Comment by pinterid** [comment]
It is supposed to be pure Django, because i prefer using it for the TASTIC monitoring tool. @schettn

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 # Django environments
 linux-venv/
 windows-env/
+env
```
Comment (schettn) [inline .gitignore]: I think that one is not really needed. Just choose windows-env or linux-venv when creating a virtual environment

**Review by schettn — CHANGES_REQUESTED** [review]
1. Have a look at my comment

2. Install black `pip3 install black` and run `black .` in the root of tastic-engine. Afterwards the whole code is formatted. Ref: [Black Formatter](https://www.google.com/search?client=safari&rls=en&q=python+black+formatter&ie=UTF-8&oe=UTF-8)
After installing Black it can also be chosen as formatter for VSCode.
3. The requirements.txt doesn't seem right. Run `pip3 freeze -> requirements.txt` to save all of your installed dependencies.

**Review by schettn — APPROVED** [review]
✅

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**